src/key-derivation/README.md

```diff
@@ -0,0 +1,9 @@
+# Key Derivation Functions
+
+[Key derivation functions] provide a way to take an input such as a password, and crate an output of
```

Comment:
crate->create

Reply:
Do we want to introduce the KDF acronym and use it?

Reply:
Thanks, fixed the typo and added the acronym